views/pwas/view.hbs

```diff
@@ -19,7 +19,7 @@
   <body>
     {{> header}}    
     <div class="section-title">
-      <a href="javascript:history.back()"><div class="back"><i class="fa fa-chevron-left" aria-hidden="true"></i></div></a>
+      <a href="{{referer}}"><div class="back"><i class="fa fa-chevron-left" aria-hidden="true"></i></div></a>
```

Comment:
Good catch, I could filter it and send them back to our home page, but I am fine sending them back to the referer as well.